schedule/yast/skip_registration/offline_install+skip_registration.yaml

```diff
@@ -3,13 +3,17 @@ name:           offline_install+skip_registration
 description:    >
   Offline installation. Skipping registration for SLE 15, as requires network connection.
   This is default behavior for SLE 12.
+vars:
+  SCC_REGISTER: ''
```

Comment:
Where do we set this setting that have to unset here? There are values like `console`, won't that do the trick?